### ISSUE TITLE
Process left open on shutdown of platform project run

### ISSUE THREAD

**Issue by GuyDaniel** [apache/netbeans#4050]
### Apache NetBeans version

Apache NetBeans 13

### What happened

After running/closing a newly built maven netbeans platform project in the ide, there was a process left running. The "Running Tasks" list in the status bar did not show any running tasks.

### How to reproduce

Create a new Java with Maven | Netbeans Application project.
Clean and build the app / branding / parent projects.
Run the project.
Exit the IDE - you will see a "Running Tasks" dialog appear.

### Did this work correctly in an earlier version?

Apache NetBeans 12.3 or earlier

### Operating System

Arch Linux 5.12.4-arch1-2

### JDK

IMPLEMENTOR="Oracle Corporation" JAVA_VERSION="17" JAVA_VERSION_DATE="2021-09-14" LIBC="gnu" OS_ARCH="x86_64" OS_NAME="Linux"

### Apache NetBeans packaging

Apache NetBeans binary zip

### Are you willing to submit a pull request?

No

### Code of Conduct

Yes

**Comment by GuyDaniel** [comment]
Yes, I can confirm that this is caused by running Netbeans with a JDK that is bundled with JFX. As soon as I switched to a standard JDK (Eg. Corretto JDK 11), the problem went away. I can also run a Platform project with a bundled JDK which shuts down ok. I haven't tried the switches at the end for the project yet.

**Comment by neilcsmith-net** [comment]
Yes, please try the exclusions at the end of that thread.  Assuming you're not using them, does excluding the `org.netbeans.html` modules fix this for you?  Just the FX one?

**Comment by GuyDaniel** [comment]
I applied the exclusion:
`                <exclusion>
                    <groupId>org.netbeans.html</groupId>
                    <artifactId>net.java.html.boot.fx</artifactId>
                </exclusion>
`
to the org.netbeans.cluster:platform artifact, but got a failure in the build in the goal _org.apache.netbeans.utilities:nbm-maven-plugin:4.3:cluster-app_:

**Some included modules/bundles depend on these codenamebases but they are not included. The application will fail starting up. The missing codenamebases are:
   net.java.html.boot.fx          ref: [org.netbeans.modules.templatesui, org.netbeans.modules.htmlui]**

**Comment by jmborer** [comment]
I noticed however that when I start an NB RCP app through Maven and try to close the IDE while the app is still running, NB believes that there are 2 processes that have been started
![unnamed](https://user-images.githubusercontent.com/6607350/170860519-39fd1497-98c7-4015-9acd-1c62d6917b1f.png)

There is Maven and the application itself, true, but it should be considered as a single process. In the status bar, there is only one process running.

**Comment by jmborer** [comment]
I tried as well 

        <dependency>
            <groupId>org.netbeans.cluster</groupId>
            <artifactId>platform</artifactId>
            <version>${netbeans.version}</version>
            <type>pom</type>
            <exclusions>
                <exclusion> 
                    <groupId>org.netbeans.html</groupId> 
                    <artifactId>net.java.html.boot.fx</artifactId> 
                </exclusion>
            </exclusions>
        </dependency>
        
        <dependency> 
            <groupId>org.netbeans.external</groupId> 
            <artifactId>net-java-html-boot-fx</artifactId> 
            <version>RELEASE113</version> 
        </dependency>

My app builds as well, but i still have the same issues with the number of processes detected:
![image](https://user-images.githubusercontent.com/6607350/170860878-9b2437d4-0518-4efe-b949-c89f6de82bba.png)

![image](https://user-images.githubusercontent.com/6607350/170860891-5c95f308-a752-4204-b639-4793100f6364.png)

**Comment by jmborer** [comment]
I retried the same exclusions with NB 16. It seems to work. No more "running" processes.

**Comment by GuyDaniel** [comment]
I have just confirmed that this is still an issue with netbeans 18. To create the problem, tell your netbeans environment to use a zulu jdk bundled with jfx, create a vanilla rcp project, then run it and stop it. You will find no process in the process bar below, however when netbe
![NB18-problem](https://github.com/apache/netbeans/assets/23487051/689f02b3-bee8-49d6-a328-f9b79356313c)
ans is exited you will find multiple of these things that have to be exited before the ide will exit:

**Comment by mbien** [comment]
Just to be clear: this is about running NB _the IDE_ on a JDK which bundles JavaFX?

i mean. We don't test this configuration. NetBeans itself does also bundle JavaFX for some features i believe.

So it doesn't surprise me that this can cause problems. We could probably try to disable the javafx modules via JVM flags and see if it makes a difference? Or just ask users to run NB on regular JDKs?
